app/views/training_modules/show.html.slim

```diff
@@ -16,30 +16,11 @@ h2.govuk-heading-m Your module progress
 p.govuk-body You can continue your learning, look at your module progress, or revisit pages of the training.
 h3.govuk-heading-s Module topics
 
-/ -------------------------------------
 .govuk-grid-row
   .govuk-grid-column-full
     hr.govuk-section-break.govuk-section-break--m.govuk-section-break--visible
     
     - @module_progress.sections.each do |section|
-      section.module-overview--section
-        .progress-bar
-          span.icon class=section[:icon]
-            span.number = section[:position]
-          - unless section[:position] == @module_progress.sections.size
-            .line
-
-        .module-section--content id="section-content-#{section[:position]}"
-          h4.govuk-heading-s= section[:heading]
-          p.govuk-body= 'This section of this module includes:'
-          .module-section--container
-            - section[:subsections].each do |mod, sub_heading, next_item, status|
-              span.module-section--item
-                - if next_item
-                  = govuk_link_to sub_heading, training_module_content_page_path(mod, next_item)
-                - else
-                  = sub_heading
-              span.progress-indicator = progress_indicator(status)
-          hr.govuk-section-break.govuk-section-break--m.govuk-section-break--visible
+      = render 'shared/module_overview_section', section: section
```

Comment:
To somewhat simplify the partial, you could see if you like the look of it with the section hash splatted in as locals.